### PR TITLE
ProjectContentV2: Implement paging for findCard

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Part of #54 

Based on 
* https://graphql.org/learn/pagination/
* https://relay.dev/graphql/connections.htm

### CODE REVIEW

File: lib/ProjectContent.ts [hunk 1]
```diff
@@ -153,40 +153,17 @@ export class ProjectContentV2 extends ProjectContent {
...
-    // FIXME: Pagination - replace "first:100" with paging. There will be more than 100 issues
-    console.log("FIXME pagination");
-    const {
-      organization: {
-        projectV2: {
-          items
+    let hasNextPage = true;
+    let lastCursor = "";
+    while (hasNextPage) {
+      const items = await this.loadPaginatedCardItems(lastCursor);
```
Comment (pavel-mikula-sonarsource) [inline lib/ProjectContent.ts:159]: Is there a better way to decompose this other than the parenthesis wrap? That's very confusing.
```
;({nodes, pageInfo: hasNextPage, endCursor} = await this.loadPaginatedCardItems(endCursor));
```

File: lib/ProjectContent.ts [hunk 2]
```diff
@@ -153,40 +153,17 @@ export class ProjectContentV2 extends ProjectContent {
...
-    // FIXME: Pagination - replace "first:100" with paging. There will be more than 100 issues
-    console.log("FIXME pagination");
-    const {
-      organization: {
-        projectV2: {
-          items
+    let hasNextPage = true;
+    let endCursor = "";
+    while (hasNextPage) {
+      const items = await this.loadPaginatedCardItems(endCursor);
```
Comment (pavel-mikula-sonarsource) [inline lib/ProjectContent.ts:159]: Is there a better way to decompose this other than the parenthesis wrap? That's very confusing.
```
;({nodes, pageInfo: hasNextPage, endCursor} = await this.loadPaginatedCardItems(endCursor));
```

Comment (ilia-kebets-sonarsource) [reply lib/ProjectContent.ts:159]: Wouldn't using `const` work?

```js
const {nodes, pageInfo: { hasNextPage, endCursor } } = await this.loadPaginatedCardItems(endCursor);
```

**Review by ilia-kebets-sonarsource — APPROVED** [review]
See my reply to your comment, otherwise LGTM!

File: lib/ProjectContent.ts (hunk 2 above)
Comment (pavel-mikula-sonarsource) [reply lib/ProjectContent.ts:159]: No, because `while (hasNextPage)` needs to read that variable outside it's scope.

**Pull request merged**